tutorials/openmm-torch-nnpops.ipynb

```diff
@@ -82,11 +82,17 @@
       },
       "outputs": [],
       "source": [
-        "# NOTE: PyTorch pinned because https://github.com/conda-forge/openmm-torch-feedstock/issues/20\n",
-        "!mamba install -q -c conda-forge \\\n",
-        "               openmm-torch nnpops torchani openmmtools \\\n",
-        "               pytorch=1.11 \\\n",
-        "               &> /dev/null # Comment this line to see a log\""
+        "# Note: at the moment only the beta releases work correctly.\n",
+        "!mamba install -q \\\n",
+        "               -c conda-forge/label/openmm_rc \\\n",
+        "               -c conda-forge/label/openmm-torch_rc \\\n",
+        "               -c conda-forge \\\n",
+        "               openmm==8.0.0beta \\\n",
+        "               openmm-torch==1.0beta \\\n",
+        "               nnpops==0.2 \\\n",
+        "               torchani==2.2.2 \\\n",
+        "               openmmtools \\\n",
+        "               &> /dev/null # Comment this line to see a log"
```

Comment:
This should be good to go with a some changes here, right?
I guess removing the channels and the betas should do the trick, also nnpops to 0.5?